package.json

```diff
@@ -20,6 +20,7 @@
   "dependencies": {
     "alexa-app": "^2.4.0",
     "alexa-verifier": "0.0.5",
```

Comment:
We still need an explicit verifier dependency? Maybe not lock to a version and let avm bring it?